magda-migrator-content-db/sql/V1_10__change_org_name.sql

```diff
@@ -1,4 +1,3 @@
-UPDATE public.content SET id='lang/en/global/appName' WHERE id='lang/en/appName';
```

Comment:
Without this line won't search.data.gov.au get its `<title>` changed to MAGDA? 😬

Reply:
On second thought I am wrong 😬